### ISSUE TITLE
Packages are copied to project library and global cache using default settings

### ISSUE THREAD

**Issue by alketh** [rstudio/renv#885]
Hey there,

thanks so much for the awesome package.

I am working on a Linux server (Debian GNU/Linux 9 (stretch), with R 3.4. The global cache and root are set to a writable network drive

`Sys.setenv(RENV_PATHS_ROOT = "/workspace_images/src/contrib")`

However, everytime I run 

`renv::init()`

the packages are copied to the global cache (as expected) but they are also copied to the project library (/renv/library). There are no error/warning indicating an abnormal behaviour.

**Comment by kevinushey** [comment]
Can you confirm what the output is if you try to symlink a dummy folder from your network drive to your project folder? For example, something like (please tweak as appropriate):

```
dir.create("//workspace_images/test")
file.symlink("//workspace_images/test", "test")
```

Is the symlink created successfully? Does R return a `TRUE` success value?

**Comment by kevinushey** [comment]
I'll also think about a way to make auditing these cases in `renv` a little easier.

**Comment by alketh** [comment]
Hey there, 

`dir.create("/workspace_images/test")
file.symlink("/workspace_images/test", "test")
[1] TRUE`

Seems to work fine on my end.

PS: Sorry for the delayed answer.

**Comment by adayim** [comment]
I am having the same issue with Windows 10. Projects and global path are live on the same network drive. The filesystem is NTFS. I saw the answer [here](https://community.rstudio.com/t/using-renv-on-network-drive-all-packages-are-copied-both-in-project-library-and-cache/83265/4) says that the junction point does not support network drive. 

I tried this on locally, it works perfectly without copying. Is there any way to work around the network drive issue?
